libbindgen/src/lib.rs

```diff
@@ -504,13 +504,36 @@ impl<'ctx> Bindings<'ctx> {
     ///
     /// Deprecated - use a `Builder` instead
     #[deprecated]
-    pub fn generate(options: BindgenOptions,
+    pub fn generate(mut options: BindgenOptions,
                     span: Option<Span>)
                     -> Result<Bindings<'ctx>, ()> {
         let span = span.unwrap_or(DUMMY_SP);
 
+        // TODO: Make this path fixup configurable?
```

Comment:
Probably you can move the `has_target_arg` to wrap the `Clang::find` block, so that it is somehow configurable?